bindings/py/tests/sparse_binary_matrix_test.py

```diff
@@ -1522,15 +1521,14 @@ def testPickling(self):
 
     a = self.Matrix.__class__(1)
     a.fromDense(mat)
+####### TODO: 
```

Comment:
all math/Sparse*Matrix classes could soon be obsoleted by #153 , so you can prioritize work on other parts first

Reply:
@ctrl-z-9000-times do you have an ETA for #153 ? As it could unbind our dependency on math/SparseMatrix, and we could continue further with #104 . If it's soon enough, @dkeeney can skip wasting time on that porting work.